.github/workflows/main.yml

```diff
@@ -83,3 +83,25 @@ jobs:
       run: cross build -vv --target ${{ matrix.platform.target }}
       working-directory: test-crate
       if: ${{ !matrix.platform.test }}
+
+  ios_cross_compile_test:
+    name: Test Cross Compile - ${{ matrix.platform.target }}
+    needs: [ test ]
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - target: aarch64-apple-ios
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        target: ${{ matrix.platform.target }}
+    - name: build
+      run: cargo build -vv --target ${{ matrix.platform.target }}
+      working-directory: test-crate
+      env:
+        # If this isn't specified the default is iOS 7, for which zlib-ng will not compile due to the lack of thread-local storage.
+        IPHONEOS_DEPLOYMENT_TARGET: 16
```

Comment:
Just curious: does this mean iOS 16 would be the minimum supported version?

Reply:
For this compilation, yes. I didn't feel like actually trying to figure out what the minimum version of iOS that `zlib-ng` could build for was, so I just used iOS 16 since that's the current SDK. This won't affect any users of the crate, it's just for compiling `zlib-sys` in CI here.